glom/test/test_path_and_t.py

```diff
@@ -168,6 +168,22 @@ def test_path_items():
     assert Path().items() == ()
 
 
+def test_path_star():
+    val = {'a': [1, 2, 3]}
+    assert glom(val, 'a.*') == [1, 2, 3]
+    val['a'] = [{'b': v} for v in val['a']]
+    assert glom(val, 'a.*.b') == [1, 2, 3]
+    assert glom(val, Path(T['a'], '*', T['b'])) == [1, 2, 3]
+    # multi-paths eat errors
+    assert glom(val, Path('a', '*', T.b)) == []
+    val = [[[1]]]
+    assert glom(val, '**') == [ [[1]], [1], 1]
+    val = {'a': [{'b': [{'c': 1}, {'c': 2}, {'c': 3}]}]}
```

Comment:
```suggestion
    val = {'a': [{'b': [{'c': 1}, {'c': 2}, {'d': {'c': 3}}]}]}
```
Will break other tests, but will test levels better with `**`

Reply:
I like it -- that also adds a negative case that `a.*.b.*.c` shouldn't see the `3`